manifest-c9s.yaml

```diff
@@ -21,7 +21,7 @@ repos:
 # Eventually we should try to build these images as part of the c9s composes.
 # In that case, the versioning should instead be exactly the same as the pungi
 # compose ID.
-automatic-version-prefix: "9.<date:%Y%m%d%H%M>"
```

Comment:
I know CentOS stream doesn't have point releases but it would be really nice to have the RHEL and C9S eqivalent builds have the same number of components in their version. I wonder if make the c9s one have a `X` in the component would be unpalatable.

Reply:
`"9.X.<date:%Y%m%d>"`

Reply:
Actually, the format I suggested here isn't semver-compatible. It needs to be three components. Letters are not allowed either, so let's just go with `0`? It's awkward but, any randomly large number feels too arbitrary.

Reply:
ok sounds good

Reply:
another option (not sure how much I prefer one or the other) is:

- centos stream: `9.<date:%Y%m%d>.spin`
- rhel 9.6: `96.<date:%Y%m%d>.spin`

where `spin` is the number of times we've had a build that day instead of `-spin` like we have now.

Reply:
I'd like to get away from the "squashed" version numbering that we've been doing and have version strings that look more natural. Not just for optics, but also e.g. it makes parsing it easier. So I think I prefer the previous option.